gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/pages/PermissionsPage.java

```diff
@@ -329,6 +331,7 @@ public String getIdValue(final Long l, final int index) {
 				// set selected or first item
 				categoryChooser.setModelObject((permission.getCategoryId() != null) ? permission.getCategoryId() : categoryIdList.get(0));
 				categoryChooser.setNullValid(false);
+				categoryChooser.setVisible(categoriesEnabled);
 				item.add(categoryChooser);
```

Comment:
Lastly, this change hides the category dropdown list from the permissions form when categories are disabled.